common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -440,7 +453,7 @@ class @Problem
         @el.addClass 'showed'
         @el.find('.show').attr('disabled', 'disabled')
         @updateProgress response
-        window.SR.readElts(answer_text)
+        window.SR.readText(gettext('The answer to the problem has been shown. Navigate through the problem to read the answer text.'))
```

Comment:
@cahrens @cptvitamin when a learner uses a screen reader to review the problem after selecting "Show Answer", are the answer texts labelled as "Answer"? Would be good to use the same terminology in the message.   
I'm wrestling with singular/plural, but how about:
"Answers to this problem are now shown. Review the problem with the answers revealed."
or "Answers to this problem are now shown. Navigate through the problem to review answers." (or "review answer text" if preferred because closer to SR labels)